ckanext/canada/templates/header.html

```diff
@@ -0,0 +1,188 @@
+{% ckan_extends %}
+
+{% set lang = h.lang() %}
+
+{% macro menu_link(r_type) %}
+  {% set label = h.recombinant_get_geno(r_type).shortname or h.recombinant_get_geno(r_type).title %}
+  <li role="presentation"><a role="menuitem" tabindex="-1" href="{{ h.url_for('recombinant.type_redirect', resource_name=r_type) }}"><i class="fa fa-upload"></i>{{ _(label) }}</a></li>
+{% endmacro %}
+
+{% block header_wrapper %}
+<header role="banner">
+ <div id="wb-bnr" class="container">
+    {%- block languagetoggle -%}
+      {% snippet 'snippets/wet-boew/languagetoggle.html' %}
+    {%- endblock -%}
+    <div class="row">
+      {% block brand %}
+        {% snippet 'snippets/wet-boew/brand.html' %}
+      {% endblock %}
+      {% block sitesearch %}
+        {% if not g.is_registry %}
+          {# Portal #}
+          <section id="wb-srch" class="col-xs-6 text-right visible-md visible-lg">
+          <h2>{{ _("Search") }}</h2>
+            <form action="https://www.canada.ca/{{ h.lang() }}/sr/srb.html" method="get" name="cse-search-box" role="search" class="form-inline">
+              <div class="form-group">
+                <label for="wb-srch-q" class="wb-inv">{{ _('Search website') }}</label>
+
+                <input id="wb-srch-q" list="wb-srch-q-ac" class="wb-srch-q form-control" name="q" value="" size="34"
+                      maxlength="170" placeholder="{{ _('Search Canada.ca') }}" type="search">
+                <datalist id="wb-srch-q-ac"></datalist>
+              </div>
+              <div class="form-group submit">
+                <button type="submit" id="wb-srch-sub" class="btn btn-primary btn-small" name="wb-srch-sub">
+                    <i class="fa fa-search" aria-hidden="true"></i><span class="wb-inv">{{ _('Search') }}</span></button>
+              </div>
+            </form>
+          </section>
+        {% endif %}
+      {% endblock %}
+      {% block user_logged %}
+        {% if g.is_registry %}
+          {# Registry #}
+          {% if c.userobj %}
```

Comment:
while making changes you could update the `c.`'s to `g.`'s, though I assume the `c.` compatibility code will be with ckan for a while yet